packages/opencensus-core/src/trace/sampler/sampler.ts

```diff
@@ -87,4 +71,28 @@ export class Sampler implements types.Sampler {
       return false;
     }
   }
-}
\ No newline at end of file
+}
+
+
+/** Builder class of Samplers */
+export class SamplerBuilder {
```

Comment:
+1 to having separate `Sampler` classes. This is much better.

Reply:
Agree.